### PR TITLE
Adding toroid and coil tet mesh scripts

### PR DESCRIPTION
**Pull request by eitan-weinstein**
There are two scripts here that both create Cubit geometries and then run tet meshes on them. For the toroid script, it creates a simple toroid, meshes, and then compares all of the mesh analytics with each other and outputs a .exo and a .h5m for the mesh with the highest resolution. Likewise, the coil script does much the same, but includes a for-loop to run for all 48 of the coils in the coils.step file.

### CODE REVIEW

File: tet_mesh_scripts/coils_tet_mesh_script.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    cubit.cmd(f'mesh volume {coil_num}')
+    cubit.cmd(f'quality volume {coil_num} {best_analytic} global draw mesh list')
+
+    # Export the mesh as a .exo file
+    exo_path = f"/home/eitan-weinstein/cubit/files_and_outputs/exo/coils/coils_tet_mesh_{coil_num}.exo"
+    cubit.cmd(f'export mesh "{exo_path}"')
+
+    # Convert to h5m
+    h5m_path = f"/home/eitan-weinstein/cubit/files_and_outputs/h5m/coils/coils_tet_mesh_{coil_num}.h5m"
+    h5m_conversion = sbp.run(f'mbconvert {exo_path} {h5m_path}', shell = True)
```
Comment (connoramoreno) [inline tet_mesh_scripts/coils_tet_mesh_script.py:161]: Paths. You can consider including the final export path as an additional argument, though outputting the H5M file to the working directory is fine as well.

**Review by connoramoreno — COMMENTED** [review]
Thanks @eitan-weinstein this is a strong start! This will come in very useful :) See my comments for suggested changes.

File: tet_mesh_scripts/toroid_tet_mesh_script.py
[the remark this answers is not in the document]
Comment (eitan-weinstein) [reply tet_mesh_scripts/toroid_tet_mesh_script.py:1]: Thanks for the suggestions! I think I got things into better shape in converting it all into a function, so I will submit another PR momentarily.

**Comment by gonuke** [comment]
One thought: what would this look like as an option during coil creation itself, rather than an add on after all the coils are created? The coil object will already be loaded into Cubit and much of the file management part can be simplified.

**Comment by connoramoreno** [comment]
This could definitely be incorporated in the export phase! Would likely be best to create and export the magnet mesh after the H5M export has been completed, as I believe the mesh of the magnets replaces the respective solids (though @eitan-weinstein can probably confirm or deny this).

**Comment by eitan-weinstein** [comment]
Is the coil creation happening in `magnet_coils.py`?

**Comment by connoramoreno** [comment]
It is but I'm not sure that the coil meshing should be included in `magnet_coils.py` itself. Maybe we should chat about this in a meeting to see how it all fits together. I believe the functionality would fit best outside of `magnet_coils` and be used after the DAGMC export, since I think the volume mesh of the coils replaces the respective solids, and the DAGMC export needs the solids themselves.

**Comment by gonuke** [comment]
The mesh should not (or should not have to) replace the geometry.  I don't think it matters when it happens relative to the DAGMC export.

**Pull request closed without merging**